pkg/apis/servicecatalog/types.go

```diff
@@ -191,7 +191,7 @@ type ServiceClass struct {
 
 	// Plans is the list of ServicePlans for this ServiceClass.  All
 	// ServiceClasses have at least one ServicePlan.
-	Plans []ServicePlan
+	Plans []Plan
```

Comment:
Plan -> ServiceClass
not
ServiceClass -> Plan. 

Isn't this an inversion of the current arrangement? Is there any particular justification for either direction?